### PR TITLE
Client side token authentication

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Client side token authentication

Add button for silent authentication
Revert client id and core version

[unstaged]

## Motivation and Context
Fix cookie impact on Google authentication

## How Has This Been Tested?
This is a prototype. Various tests were performed locally.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc @santiagonoguez I would like to merge this in the main branch/PR. Please review. Thanks!

File: web/partials/common-header/common-header.html [hunk 1]
```diff
@@ -67,6 +67,9 @@
 						ng-controller="CloseFrameButtonCtrl"
 						src="'partials/common-header/close-frame-button.html'"
 					></ng-include>
+          <li class="madero-style">
+            <button class="btn btn-default btn-navbar" ng-click="googleAuthFactory.authenticate(true, true)">Authenticate</button>
+          </li>
```
Comment (santiagonoguez) [inline web/partials/common-header/common-header.html:72]: weird indentation

File: web/scripts/components/userstate/services/svc-google-auth-factory.js [hunk 2]
```diff
@@ -102,30 +102,48 @@
...
+             (silent ? 'prompt=none&' : '') +
+             // 'redirect_uri=http://localhost:8000/&' +
+             'redirect_uri=' + loc + '&' +
+             // 'nonce=' + Math.floor((Math.random() * 10000) + 1) + '&' +
+             (silent ? 'login_hint=' + userState.getUsername() + '&' : '') +
+             // 'redirect_uri=https://google-oauth2-dot-rvacore-test.appspot.com/oauth2callback&' +
+             // 'client_id=614513768474-dnnhi8e6b8motn6i5if2ur05g6foskoc.apps.googleusercontent.com';
+             'client_id=614513768474.apps.googleusercontent.com';
+
+          return;
```
Comment (santiagonoguez) [inline web/scripts/components/userstate/services/svc-google-auth-factory.js:121]: @alex-deaconu curious, aren't clients expecting a promise here ?

Comment (alex-deaconu) [reply web/scripts/components/userstate/services/svc-google-auth-factory.js:121]: @santiagonoguez to confirm this is prototype code. Yes, but considering we redirect above I think it's ok.  Will fix subsequently to ensure we also use the popup auth to support those environments that require it. Thanks.

File: web/partials/common-header/common-header.html (hunk 1 above)
Comment (alex-deaconu) [reply web/partials/common-header/common-header.html:72]: Will fix in subsequent PRs.

**Review by santiagonoguez — APPROVED** [review]

**Pull request merged**